charmcraft.yaml

```
@@ -10,10 +10,22 @@ assumes:
   - juju >= 3.4.0
 
 description: |
-  Tempo is a distributed tracing backend by Grafana.
+  Tempo provides tracing endpoints for charms and a decorator to auto-instrument the 
+  charms to push traces using common opensource tracing protocols, including Jaeger,
+  Zipkin, and OpenTelemetry. Then, propagate those pushed traces into a Grafana instance.
```

I think we can keep:
```suggestion
  Tempo is a distributed tracing backend by Grafana, supporting Jaeger,
  Zipkin, and OpenTelemetry protocols.
```
Iiuc, the rest is a description of [one of the charm's lib](https://github.com/canonical/tempo-k8s-operator/blob/main/lib/charms/tempo_k8s/v1/charm_tracing.py).